Revise Microservice Transaction Sample (#35)
scalar-labs/scalardb-samples: microservice-transaction-sample/README.md

# Microservice Transaction Sample

> This tutorial describes how to create a sample application for Microservice Transaction that uses Two-phase Commit Transactions in ScalarDB.

Author: The other changes are mainly refinements in wording.

And the `repayment` endpoint of Customer Service is for transaction #5.

## Setup

### Start Cassandra and MySQL

To start Cassandra and MySQL, you need to run the following `docker-compose` command:

```shell
$ docker-compose up -d cassandra mysql
```

Please note that you need to wait around more than one minute for the containers to be fully started.

### Load schema

> You then need to apply the schema with the following commands.
> Please download the schema tool `scalardb-schema-loader-<version>.jar` that can be found in [releases](https://github.com/scalar-labs/scalardb/releases) of ScalarDB.

Author: ```suggestion
You then need to apply the schema with the following commands.
To download the schema loader tool, `scalardb-schema-loader-<VERSION>.jar`, see the [Releases](https://github.com/scalar-labs/scalardb/releases) of ScalarDB and download the version that you want to use.
```

For MySQL:

```shell

> $ java -jar scalardb-schema-loader-<version>.jar --config database-mysql.properties --schema-file customer-service-schema.json

Author: ```suggestion
$ java -jar scalardb-schema-loader-<VERSION>.jar --config database-mysql.properties --schema-file customer-service-schema.json
```

```

For Cassandra:

```shell

> $ java -jar scalardb-schema-loader-<version>.jar --config database-cassandra.properties --schema-file order-service-schema.json --coordinator

Author: ```suggestion
$ java -jar scalardb-schema-loader-<VERSION>.jar --config database-cassandra.properties --schema-file order-service-schema.json --coordinator
```
